### PR TITLE
creating image folder

### PR DESCRIPTION
**Pull request by RandyKoiSA**
created the image folder and edited the image from the README.md file.

### CODE REVIEW

**Review by scriptnull — CHANGES_REQUESTED** [review]
Hi @RandyKoiSA,

Thanks for the PR. A couple of suggestions here,

1. Could you rename `image1.png` to `graphviz.png`? (it will give more context when we have multiple images)

2. I see that you have said "edited the image from the README.md file" in the pull request description. But I couldn't find the respective changes for it. May be you forgot to commit / push the commit to your fork?

Also please note that there are couple of other PRs active for the same issue right now. So please check if they are merged before working on the suggested changes.

**Comment by RandyKoiSA** [comment]
Hi scriptnull,

Thank you for the feedback. I will get that fixed.

**Comment by RandyKoiSA** [comment]
Hi scriptnull,

Hopefully, I got it right this time.  Would be great if you can check the recent changes.

Thank you

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ digraph G {
 
 which could be used with tools like [Webgraphviz](http://www.webgraphviz.com/) to get a graphic like
 
-![image](https://user-images.githubusercontent.com/4211715/52170132-a8971100-276a-11e9-8ef3-b4cfdb494fb6.png)
+![image](/images/image1.png)
```
Comment (scriptnull) [inline README.md]: Could you update the image reference here?

```
![graphviz](/images/graphviz.png)
```

**Comment by RandyKoiSA** [comment]
I just updated the README.md again. I apologize on my mistakes, I made the change onto the master but not onto the image_folder branch.

**Review by scriptnull — APPROVED** [review]
Cool! No problem. Thanks for the effort! :+1:

**Pull request merged**